vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -253,28 +238,30 @@ type::ExternalWalletPtr GenerateLinks(type::ExternalWalletPtr wallet) {
   }
 
   switch (wallet->status) {
-    case type::WalletStatus::PENDING: {
-      wallet->add_url = GetSecondStepVerify();
-      wallet->withdraw_url = GetSecondStepVerify();
-      break;
-    }
-    case type::WalletStatus::CONNECTED: {
-      wallet->add_url = GetAddUrl(wallet->address);
-      wallet->withdraw_url = GetSecondStepVerify();
-      break;
-    }
     case type::WalletStatus::VERIFIED: {
+      DCHECK(!wallet->token.empty());
+      DCHECK(!wallet->address.empty());
       wallet->add_url = GetAddUrl(wallet->address);
       wallet->withdraw_url = GetWithdrawUrl(wallet->address);
       break;
     }
+    case type::WalletStatus::PENDING: {
+      DCHECK(!wallet->token.empty());
+      DCHECK(wallet->address.empty());
+      wallet->add_url = GetSecondStepVerify();
+      wallet->withdraw_url = GetSecondStepVerify();
+      break;
+    }
     case type::WalletStatus::NOT_CONNECTED:
-    case type::WalletStatus::DISCONNECTED_VERIFIED:
-    case type::WalletStatus::DISCONNECTED_NOT_VERIFIED: {
-      wallet->add_url = "";
-      wallet->withdraw_url = "";
+    case type::WalletStatus::DISCONNECTED_VERIFIED: {
+      DCHECK(wallet->token.empty());
+      DCHECK(wallet->address.empty());
+      wallet->add_url = {};
```

Comment:
C++ has a ridiculous amount of ways in which a variable can be initialized. Nicolai Josuttis made a matrix to summarize the rules for most of them (you can check it out [here](http://josuttis.com/cpp/c++initialization.pdf)).
I tend to stick with uniform initialization, since it is a single initialization syntax, which I can use _almost_ anywhere and I often use the same syntax when assigning a variable.
But yeah, as far as `std::string`s, `""` works, too. Compilers tend to generate somewhat smaller assembly code with `{}` though, e.g. https://godbolt.org/z/sq5368efs.